src/main/java/frc/robot/subsystems/Arm.java

```diff
@@ -4,17 +4,19 @@
 
 package frc.robot.subsystems;
 
+import edu.wpi.first.wpilibj.DutyCycleEncoder;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
 public class Arm extends SubsystemBase {
 
     private Shoulder m_shoulder;
     private Reacher m_reacher;
+    private DutyCycleEncoder m_Duty_Encoder = new DutyCycleEncoder(0);
 
     public Arm() {
         // Construct the shoulder and reacher trapezoid subsystems
-        m_shoulder = new Shoulder();
+        m_shoulder = new Shoulder(m_Duty_Encoder);
```

Comment:
Why is the DutyEncoder created in the Arm? Why not just construct it as part of the Shoulder?